src/App.tsx

```diff
@@ -7,34 +8,21 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
+const preparedTabs = tabs.map((tab, i) => ({ ...tab, index: i }));
```

Comment:
no don't use `indexes`, what if you delete one of the tabs? you already have `tab.id`
```suggestion
```